backend/src/graphql/resolver/UserResolver.test.ts

```diff
@@ -1134,6 +1134,129 @@ describe('UserResolver', () => {
+
+        describe('too long', () => {
+          it('throws and logs an error', async () => {
+            await expect(
+              mutate({
+                mutation: updateUserInfos,
+                variables: {
+                  alias: 'bibis_alias_far_too_long',
+                },
+              }),
+            ).resolves.toMatchObject({
+              errors: [new GraphQLError('Given alias is too long')],
+              data: null,
+            })
+            expect(logger.error).toBeCalledWith(
+              'Given alias is too long',
+              'bibis_alias_far_too_long',
+            )
+          })
+        })
+
+        describe('invalid characters', () => {
+          it('throws and logs an error', async () => {
+            await expect(
+              mutate({
+                mutation: updateUserInfos,
+                variables: {
+                  alias: 'no_underscore',
+                },
+              }),
+            ).resolves.toMatchObject({
+              errors: [new GraphQLError('Invalid characters in alias')],
+              data: null,
+            })
+            expect(logger.error).toBeCalledWith('Invalid characters in alias', 'no_underscore')
+          })
+        })
+
+        describe('alias exists', () => {
+          let peter: User
+          beforeAll(async () => {
+            peter = await userFactory(testEnv, peterLustig)
+            await mutate({
+              mutation: login,
+              variables: {
+                email: 'peter@lustig.de',
+                password: 'Aa12345_',
+              },
+            })
+            await mutate({
+              mutation: updateUserInfos,
+              variables: {
+                alias: 'bibiBloxberg',
+              },
+            })
+            await mutate({
+              mutation: login,
+              variables: {
+                email: 'bibi@bloxberg.de',
+                password: 'Aa12345_',
+              },
+            })
+          })
+
+          afterAll(async () => {
+            const [user] = await User.find({ id: peter.id })
+            await user.remove()
+          })
+
+          it('throws and logs an error', async () => {
+            await expect(
+              mutate({
+                mutation: updateUserInfos,
+                variables: {
+                  alias: 'bibiBloxberg',
+                },
+              }),
+            ).resolves.toMatchObject({
+              errors: [new GraphQLError('Alias already in use')],
+              data: null,
+            })
+            expect(logger.error).toBeCalledWith('Alias already in use', 'bibiBloxberg')
+          })
+        })
+
+        describe('valid alias', () => {
+          it('updates the user in DB', async () => {
+            await mutate({
+              mutation: updateUserInfos,
+              variables: {
+                alias: 'bibiBloxberg',
+              },
+            })
+            await expect(User.findOne()).resolves.toEqual(
```

Comment:
Are you sure that we will always only have one entry in ``users``?
Please search for the user that should have the alias setten to prevent eventuell flakyness.
For example like this:
```suggestion
            await expect(User.findOne({ firstname: 'Bibi', lastname: 'Bloxberg' })).resolves.toEqual(
```

Reply:
Yes, the test is setup to have only one user. Therefore the workaround with the existing alias